src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -1072,7 +1124,7 @@ void JasmineGraphFrontEnd::removeGraph(std::string graphID, SQLiteDBInterface sq
             "SELECT name, partition_idpartition FROM worker_has_partition INNER JOIN worker ON "
             "worker_has_partition.worker_idworker = worker.idworker WHERE partition_graph_idgraph = " + graphID + ";");
     for (vector<vector<pair<string, string>>>::iterator i = hostPartitionResults.begin();
-         i != hostPartitionResults.end(); ++i) {
+    i != hostPartitionResults.end(); ++i) {
```

Comment:
Better we get these for loop statements into one line format.

Reply:
Fixed